doc/source/release-notes/Percona-Server-8.0.16-7.rst

```diff
@@ -0,0 +1,95 @@
+.. rn:: 8.0.16-7
+================================================================================
+|Percona Server| |release|
+================================================================================
+|Percona| announces the release of |Percona Server| |release| on |date|
+(downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from the `Percona
+Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+This release includes fixes to bugs found in previous releases of |Percona
+Server| 8.0.
+|Percona Server| |release| is now the current GA release in the 8.0
+series. All of |Percona|’s software is open-source and free.
+
+Percona Server for MySQL 8.0 includes all the `features and bug fixes available in MySQL 8.0.16
+Community Edition
+<https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-16.html>`__ in addition to
+enterprise-grade features developed by Percona.
+
+Bugs Fixed
+================================================================================
+- Parallel doublewrite buffer writes must crash the server on an I/O error occurs. Bug fixed :psbug:`5678`.
+
+- After resetting the :variable:`innodb-temp-tablespace-encrypt` to `OFF` during runtime the subsequent file-per-table temporary tables continue to be encrypted. Bug fixed :psbug:`5734`.
```

Comment:
:variable:`innodb_temp_tablespace_encrypt`

Reply:
updated.

Reply:
I'm still seeing this as ":variable:`innodb-temp-tablespace-encrypt`" with - instead of _